uktrade/data-hub-frontend
# Combining feature flags

## Description of change

We’re planning to set up an early access alpha for users. A final bit of tidying up for this is to merge the two feature flags we currently use into one. This reduces the risk of users only seeing some changes but not all.

## Test instructions
- Bring up this branch on your local dev frontend and navigate to: 
- http://localhost:3000/events
- http://localhost:3000/events/aventri/1111/details
- http://localhost:3000/contacts/68be55fc-544d-4d9c-a264-977b917f1fd3/interactions
- Your page should look like the before screenshots below.

- In Django dev API, add the user feature flag `user-activity-stream-aventri` to your adviser profile.
- The same pages should look like the after screenshots below.
## Screenshots

### Before - Without feature flag
Events Collection Page (still all the old filters)
![image](https://user-images.githubusercontent.com/70902973/190679408-0bf8f5f4-7fa5-4f94-a7e9-6304df39b821.png)
Aventri Events Page (nothing displayed on the page)
![image](https://user-images.githubusercontent.com/70902973/190679459-6eb68040-255c-474b-bd71-f8bd4ebb9113.png)
Contact Interactions tab (no aventri activities)
![image](https://user-images.githubusercontent.com/70902973/190679617-fa25dabf-58a7-484b-8bab-6547c46598af.png)

### After - With feature flag
Event Collection Page (building up new filters - other filters being added in a new ticket)
![image](https://user-images.githubusercontent.com/70902973/190680240-24086ac7-3520-47ed-a57a-56c8f979dcef.png)
Aventri Events Page (can now see aventri event details)
![image](https://user-images.githubusercontent.com/70902973/190680528-d25d7a23-484c-4ea5-9991-0c95f8e565fd.png)
Contact Interactions tab (aventri activities visible)
![image](https://user-images.githubusercontent.com/70902973/190679975-677d9648-9181-404e-a7e6-6720d5dce661.png)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
